### ISSUE TITLE
save training and test datsets

### ISSUE THREAD

**Issue by deri0s** [deri0s/Deep-Learning#1]
- The iris dataset is ordered by the flower species.

Since there is only 150 observations, if I divide training and test datasets using the 80% and 20% of the entire data, and the test dataset is taken from the last observations, then the test data will only have samples of the species `verinica`.

- Maybe I have to shuffle the data first and then use the `train_test_split` methos from sklearn.
- Maybe the solution is more simpler than that and I just have to use the `train_test_split` method using an option that always gives me the same shuffle.
